src/S3/s3sound.c

```diff
@@ -55,10 +55,10 @@ int S3LoadSample(tS3_sound_id id) {
     if (memcmp(buf, "RIFF", 4) == 0) {
         wav_header* hdr = (wav_header*)buf;
         sample->dataptr = &buf[sizeof(wav_header)];
-        sample->size = hdr->data_bytes;
-        sample->rate = hdr->sample_rate;
-        sample->resolution = hdr->bit_depth;
-        sample->channels = hdr->num_channels;
+        sample->size = BrSwap32(hdr->data_bytes);
+        sample->rate = BrSwap32(hdr->sample_rate);
+        sample->resolution = BrSwap16(hdr->bit_depth);
+        sample->channels = BrSwap16(hdr->num_channels);
```

Comment:
Is an `#ifdef` missing here?
```suggestion
#if BR_ENDIAN_BIG
        sample->size = BrSwap32(hdr->data_bytes);
        sample->rate = BrSwap32(hdr->sample_rate);
        sample->resolution = BrSwap16(hdr->bit_depth);
        sample->channels = BrSwap16(hdr->num_channels);
#else
        sample->size = hdr->data_bytes;
        sample->rate = hdr->sample_rate;
        sample->resolution = hdr->bit_depth;
        sample->channels = hdr->num_channels;
#endif
```

Reply:
Oops, indeed. I got these mixed up with BrLtoHL and BrLtoHS.